### ISSUE TITLE
Error in CashBackInfo class_dict

### ISSUE THREAD

**Issue by grumpy-curmudgeon** [sidecars/python-quickbooks#45]
In deposits.py, line 28:

```
    class_dict = {
        "AccountRef": None
    }
```

should read:

```
    class_dict = {
        "AccountRef": Ref
    }
```

as currently written, when trying to process a deposit, it attempts to instantiate a None object instead of a Ref during from_json() (also) line 28:

`                 sub_obj = obj.class_dict[key]()`

**Comment by ej2** [comment]
Good catch. If you want to submit a PR, I will merge it in.

**Comment by ej2** [comment]
Fixed in version 0.5.2
